jcl/source/windows/JclDebug.pas

```diff
@@ -3498,6 +3545,73 @@ function TJclBinDebugScanner.SourceNameFromAddr(Addr: DWORD): string;
     Result := '';
 end;
 
+function TJclBinDebugScanner.VAFromUnitAndProcName( UnitName, ProcName : string ): DWORD;
+var
+  P          : Pointer;
+  VA         : Cardinal;
+  i,
+  Value,
+  FirstWord,
+  SecondWord : Integer;
+  S          : string;
```

Comment:
Do not indent-align type names
No space before a colon
variable names start with a capital letter `i` -> `I`